frontend/src/components/maps/leaflet/InfoSlideOut/InfoSlideOut.tsx

```diff
@@ -138,36 +130,14 @@ const InfoControl: React.FC<InfoControlProps> = ({ open, setOpen, onHeaderAction
           pathname: `/mapview`,
           search: queryString.stringify({
             ...queryString.parse(location.search),
+            sidebar: true,
+            disabled: true,
+            loadDraft: false,
+            buildingId: 0,
+            associatedParcelId: propertyInfo?.id,
+            parcelId: undefined,
           }),
         }}
-        onClick={() => {
```

Comment:
I ended up not liking this because there is no way to load an associated building form from a url, so I refactored this to allow for that.